paymentsheet/src/main/java/com/stripe/android/customersheet/data/CustomerSheetSession.kt

```diff
@@ -0,0 +1,9 @@
+package com.stripe.android.customersheet.data
+
+import com.stripe.android.model.ElementsSession
+import com.stripe.android.model.PaymentMethod
+
+internal data class CustomerSheetSession(
+    val elementsSession: ElementsSession,
+    val paymentMethods: List<PaymentMethod>
+)
```

Comment:
Ended up naming this `CustomerSheetSession`. There will be additional attributes added when feature sets are introduced later